app/services/report_importer.rb

```diff
@@ -33,13 +33,14 @@ def initialize(raw, proxy_id = nil)
   end
 
   def import
-    start_time = Time.now.utc
-    logger.info "processing report for #{name}"
-    logger.debug { "Report: #{raw.inspect}" }
+    start_time = Time.now
+    logger.debug { "Processing report: #{raw.inspect}" }
     create_report_and_logs
     if report.persisted?
-      logger.info("Imported report for #{name} in #{(Time.now.utc - start_time).round(2)} seconds")
-      host.refresh_statuses
+      imported_time = Time.now
+      host.refresh_statuses(statuses_for_refresh)
```

Comment:
This line is the only change in this method, the rest are changes in logging so it produces saner output.